IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcGeoScienceObservationTypeEnum/Documentation.md

```diff
@@ -0,0 +1 @@
+This container defines the different predefined types of geoscience observation types that can specify an _IfcGeoScienceObservation_.
```

Comment:
Is "container" the right term? I guess it should be "enumeration".

Reply:
Done.